src/elf-create-argp.c

```diff
@@ -45,20 +50,57 @@ int parse_arguments(int argc, char *argv[], elf_create_args *arguments)
 		return -1;
 	}
 
+	if (arguments->exports && entrypoint_list)
+	{
+		printf("Options -m and -e cannot be used together\n");
+		return -1;
+	}
+
 	if (arguments->exports && arguments->exports_output)
 	{
 		printf("Options -g and -e cannot be used together\n");
 		return -1;
 	}
-	
+
+	if (entrypoint_list)
+	{
+		char *entrypoint = entrypoint_list;
+		char *entrypoint_end = strstr(entrypoint, ",");
+		int i = 0;
+		while (entrypoint_end != NULL)
+		{
+			if (i >= 2)
+			{
+				fprintf(stderr, "list for -m too long\n");
+				return -1;
+			}
+			size_t entrypoint_length = (size_t)(entrypoint_end - entrypoint);
+			if (entrypoint_length > 0)
+				arguments->entrypoint_funcs[i] = strndup(entrypoint, entrypoint_length);
```

Comment:
Where is this `free` code? ~~Well, it is not required because all memory will be freed when this process ends.~~

Reply:
The pointers are passed directly to the vita_export_t fields, and so they'll be freed by vita_exports_free